golden/size-of-pactvalue-literal/golden

```diff
@@ -0,0 +1 @@
+[226,"V+I_x-~=V*~jyvRk3fIuCKU0c/K𠽉O#pn$t#|JddYzspfH?04q𤚻FcldzeS|𧘛$Q%7vd/?𨰁𐎿O$p7"]
```

Comment:
This is minimal coverage, and I think pact value literals are well-covered no? If so remove this golden.

Reply:
See previous comment.